jdbc-driver/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
It seems that we still need to keep it even though Cluster is no longer mocked... otherwise the tests will fail. I suspect it is because we have `when(mockListDefault.iterateAll())
                .thenReturn(Collections.singleton(cluster2))
                .thenReturn(Collections.singleton(cluster1));`